koku/api/settings/test/tags/test_api.py

```diff
@@ -133,16 +151,21 @@ def test_tags_order_by(self):
             self.assertEqual(important_values, test_case["expected"])
 
     def test_tags_order_by_invalid(self):
+        test_cases = (
+            {"description": "Standard DRF syntax", "params": {"order_by": "NOPE"}},
+            {"description": "Custom order_by syntax", "params": {"order_by[NOPE]": "asc"}},
```

Comment:
Does this API support two syntaxes; `order_by=key` and `order_by[key]`?

Reply:
Yes. The default syntax, `order_by=field` we get for free from Django/DRF. Supporting the dictionary syntax, `order_by[key]`, requires extra work.